server/utils/auth.py

```diff
@@ -0,0 +1,117 @@
+from os import abort
+from flask import redirect, request, session, url_for, render_template
+from flask_login import LoginManager, login_user, logout_user, login_required
+from flask_oauthlib.client import OAuth
+from oauth2client.contrib.flask_util import UserOAuth2
+
+
+from server import app
+from server.models import db, User
+import server.utils.canvas as canvas_client
+from server.utils.stub import get_dev_user_oauth_resp
+from server.form import DevLoginForm
+
+login_manager = LoginManager(app=app)
+
+oauth = OAuth()
+
+server_url = app.config.get('CANVAS_SERVER_URL')
```

Comment:
this should be canvas_server_url, otherwise its ambigous

Reply:
true